UnityProject/Assets/Scripts/Player/IPlayerPossessable.cs

```diff
@@ -143,7 +143,7 @@ public void InternalOnPossessPlayer(Mind mind, IPlayerPossessable parent)
 		{
 			possessing.InternalOnPossessPlayer(mind, this);
 		}
-		OnPossessedBy?.Invoke(mind, parent);
+		OnPossessedBy.Invoke(mind, parent);
```

Comment:
dont

put it back

Reply:
the issue was, it's supposed to be initialised by the implementation, and if it is not it set up incorrectly

Reply:
this will cause issues if left as is

Reply:
it's fine though if you implement the interface properly it will work, if you don't implement it properly it will error and it's an indication that you have it set up Incorrectly

Reply:
the elvis on events is making sure it isn't invoked if it has no subscribers at that point. There is literally zero reason to remove it

Reply:
it's a unity event so it doesn't pop into existence when you subscribe into it , It has to be made by the class initialiser and then you can subscribe to it

Reply:
at any point in the game whatever object subscribed to it can die, then you have no subscribers and you are still invoking the event. With the elvis operator it won't be invoked when it has no subscribers

Reply:
this tbh